srcpkgs/cgo/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'cgo'
+pkgname=cgo
+version=0.5.0
+revision=1
+archs="x86_64* i686*"
```

Comment:
This is not right because the program can certainly be compiled for most if not all target architectures.
I think you'd just have to remove the line `CC = cc` from `Makefile` to make cross building work.

Reply:
ok, I looked others template, to get the idea, hope it's ok now.